android/app/src/androidTest/java/com/microsoft/codepush/common/managers/SettingManagerAndroidTests.java

```diff
@@ -115,9 +115,9 @@ public void pendingUpdateIsNull() throws Exception {
     @Test
     public void failedUpdateCompatibilityTest() throws Exception {
         mSettingsManager.removeFailedUpdates();
-        CodePushLocalPackage codePushLocalPackage = createLocalPackage(PACKAGE_HASH);
+        CodePushPackage codePushLocalPackage = createPackage(PACKAGE_HASH);
         CommonSettingsCompatibilityUtils.saveFailedUpdate(mCodePushUtils.convertObjectToJsonObject(codePushLocalPackage), InstrumentationRegistry.getContext());
-        List<CodePushLocalPackage> codePushLocalPackages = mSettingsManager.getFailedUpdates();
+        List<CodePushPackage> codePushLocalPackages = mSettingsManager.getFailedUpdates();
```

Comment:
Why have we switched from  `CodePushLocalPackage` to `CodePushPackage`?

Reply:
@sergey-akhalkov because that was the logical mistake I previously fixed in the main code, but but not in tests.